report/41_Lessons-Evolution-&-Refactoring.md

```diff
@@ -66,7 +66,8 @@ The team knew that choosing scalable techonologies early was important.
 If a choice to switch from a monolithic server to a distributed system comes too late, then the changes that will have to be made could become quite big, resulting in a lot of technical debt.
 
 This was why dockerization came early for the project, with separation between components for easier clusterization.
-The team didn't quite get Docker Swarm to work, but because these considerations were done early, Docker Swarm was actually obtainable, instead of being buried in debt from not having a RESTful API, for example.
+Because the considerations were done early, Docker Swarm was actually obtainable, instead of being buried in debt from not having a RESTful API, for example.
+The team unfortunately didn't get Docker Swarm to work though the work put into trying can be found on the `swarm` branch.
```

Comment:
Using "The Team" makes for a awkward sentence. 
The problem with "we" is not the word, but the way it structures the sentence.